openassessment/templates/openassessmentblock/peer/oa_peer_assessment.html

```diff
@@ -1,6 +1,7 @@
 {% load tz %}
 {% load i18n %}
 {% block list_item %}
+{% load staticfiles %}
 <li
```

Comment:
This staticfile load is a remnant from a previous iteration (dateutils in edx-platform uses a static file loader). Will delete.